### ISSUE TITLE
Update jaeger-core to depend gson 2.8.5

### ISSUE THREAD

**Issue by philipzhaoTS** [jaegertracing/jaeger-client-java#580]
Is there a plan to update the jaeger-core to depend on gson 2.8.5. Currently, it is depending on gson 2.8.2.

**Comment by vprithvi** [comment]
While we can update this, I'd like to understand more about the intent of your question. 
Does the current dependency cause any problems for you?

**Comment by philipzhaoTS** [comment]
Correct. Some of our codebase is using gson2.8.5 and this will lead to build conflict.

**Comment by philipzhaoTS** [comment]
Ping, is there a plan to upgrade to newer version of gson?

**Comment by yurishkuro** [comment]
would you like to create a PR?
